terraform/terraform.tf

```diff
@@ -0,0 +1,50 @@
+terraform {
+  required_version = "~> 0.13.5"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "2.29.0"
+    }
+    cloudfoundry = {
+      source  = "cloudfoundry-community/cloudfoundry"
+      version = "0.12.6"
+    }
+    statuscake = {
+      source  = "terraform-providers/statuscake"
+      version = "1.0.0"
+    }
+  }
+  backend "azurerm" {
+  }
+}
+
+provider cloudfoundry {
+  api_url      = local.cf_api_url
+  user         = var.cf_user != "" ? var.cf_user : null
```

Comment:
I found we can simplify by setting the default valut to `null` in variable section. See:
- https://github.com/DFE-Digital/teacher-vacancy-service/blob/terraform-monitoring-0-13-4/terraform/monitoring/variables.tf#L1
- https://github.com/DFE-Digital/teacher-vacancy-service/blob/terraform-monitoring-0-13-4/terraform/monitoring/provider.tf#L4